--- b/arch/arm64/boot/dts/broadcom/stingray/Makefile
+++ a/arch/arm64/boot/dts/broadcom/stingray/Makefile
@@ -21,7 +21,7 @@
 dtb-$(CONFIG_ARCH_BCM_IPROC) += bcm958742t-paxb-iommu.dtb
 dtb-$(CONFIG_ARCH_BCM_IPROC) += bcm958742t-pcie-iommu.dtb
 
-dtb-$(CONFIG_ARCH_BCM_IPROC) += bcm958802a8023.dtb
+# dtb-$(CONFIG_ARCH_BCM_IPROC) += bcm958802a8023.dtb
 
 dtb-$(CONFIG_ARCH_BCM_IPROC) += bcm958802a804x.dtb
 
